File: src/ProjectR.Scripting/content/scripts/spells/lua/Untouchable.lua
function GetTargetType()
    return Myself
end

function GetName()
return "Untouchable"
end

function GetDescription()
  return "Your concentration allows you to better react to incoming attacks.\nDoubles EVA."
end

function IsSupportSpell()
    return false
end

function GetMPCost()
    return .24
end

function GetDelay()
  return .66
end

function GetSpellType()
  return Magical
end

function GetMasteries()
  return {  }
end

function SpellEffect()
 a:BuffStat(EVA, 1)
end

File: src/ProjectR.Scripting/content/scripts/spells/Untouchable.cs
using System.Collections.Generic;
using ProjectR.Interfaces;
using ProjectR.Interfaces.Model;
using ProjectR.Interfaces.Model.Stats;

namespace ProjectR.Scripting.Spells
{
    public class Untouchable : SpellScriptBase
    {
        public override string Name { get { return "Untouchable"; } }
        public override string Description { get { return "Your concentration allows you to better react to incoming attacks.\nGreatly increases EVA."; } }

        public override TargetType TargetType { get { return TargetType.Myself; } }
        public override IList<EleMastery> Masteries { get { return new EleMastery[] { }; } }
        public override SpellType SpellType { get { return SpellType.Magical; } }
        public override bool IsSupportSpell { get { return false; } }

        public override double MPCost { get { return 24; } }
        public override double Delay { get { return .66; } }

        public override void SpellEffect(ICharacter caster)
        {
            BuffStat(Stat.EVA, 1);
        }
    }
}